common/models/access-token.js

```diff
@@ -171,7 +171,8 @@ module.exports = function(AccessToken) {
     cookies = cookies.concat(['access_token', 'authorization']);
 
     for (length = params.length; i < length; i++) {
-      id = req.param(params[i]);
+      // Only look up params in query. There is no body here.
+      id = req.query && req.query[params[i]];
```

Comment:
And, really, unless this is used outside of Express, `req.query` is guaranteed to be an object.